.github/workflows/build-and-test.yml

```diff
@@ -143,6 +143,7 @@ jobs:
                   path: dist
 
             - uses: actions/upload-artifact@v2
+              continue-on-error: true
```

Comment:
Since that timeout issue seems to have been a one time thing, I'm not sure we need this?

Reply:
It happened a few times for me before I added this. Plus I noticed that all the other steps that deal with diffing the conda envs already have `continue-on-error`, which I think is the right thing to do since this stuff is not essential.